lib/taurus/core/taurusmanager.py

```diff
@@ -108,27 +112,59 @@ def cleanUp(self):
 
         self._thread_pool.join()
         self._thread_pool = None
+        self._sthread_pool.join()
+        self._sthread_pool = None
 
         self._state = ManagerState.CLEANED
 
     def addJob(self, job, callback=None, *args, **kw):
-        """Add a new job (callable) to the queue. The new job will be processed
-        by a separate thread
+        """ Deprecated. Wrapper of enqueueJob. See enqueueJob documentation.
+        """
+        self.deprecated(dep='addJob', alt='enqueueJob', rel='4.3.2')
+        self.enqueueJob(job, callback, args, kw)
 
+    def enqueueJob(self, job, callback=None, jobargs=None, jobkwargs=None,
```

Comment:
PEP8: change jobargs to job_args and jobkwargs to job_kwargs

Note that this implies changes everywhere else where it is used!